pimd/pim6_cmd.c

```diff
@@ -146,6 +146,46 @@ DEFUN (interface_no_ipv6_mld_join,
     return nb_cli_apply_changes(vty, NULL);
 }
 
+DEFUN (interface_no_ipv6_mld_version,
+       interface_no_ipv6_mld_version_cmd,
+       "no ipv6 mld version (1-2)",
+       NO_STR
+       IPV6_STR
+       IFACE_MLD_STR
+       "MLD version\n"
+       "MLD version number\n")
+{
+    nb_cli_enqueue_change(vty, "./mld-version", NB_OP_DESTROY, NULL);
+    return nb_cli_apply_changes(vty, FRR_GMP_INTERFACE_XPATH,
+                    "frr-routing:ipv6");
+}
+
+DEFUN (interface_ipv6_mld_query_interval,
```

Comment:
this commit is saying it is for mld version but here you have added query-interval